recipes/configs/qwen2_5/14B_to_7B_KD_lora_single_device.yaml

```diff
@@ -0,0 +1,148 @@
+# Config for single device knowledge distillation (KD) in knowledge_distillation_single_device.py
+# using a Qwen14B teacher and student model
+#
+# This config assumes that you've ran the following commands before launching KD:
+# First download the student and teacher models
+#   tune download Qwen/Qwen2.5-14B-Instruct --output-dir /tmp/Qwen/Qwen2.5-14B-Instruct --ignore-patterns "original/consolidated.00.pth"
+#   tune download Qwen/Qwen2.5-7B-Instruct --output-dir /tmp/Qwen/Qwen2.5-7B-Instruct --ignore-patterns "original/consolidated.00.pth"
+#
+# You get better results using KD if the teacher model has already been fine-tuned on the target dataset:
+#   tune run lora_finetune_single_device --config qwen2_5/14B_lora_single_device.yaml
+#
+# To launch on a single device, run the following command from root:
+#   tune run knowledge_distillation_single_device --config qwen2_5/14B_to_7B_KD_lora_single_device.yaml
+#
+# This config works only for training on single device.
+
+
+output_dir: /tmp/kd_7B # /tmp may be deleted by your system. Change it to your preference.
+
+# Model Arguments
+model:
+  _component_: torchtune.models.qwen2_5.lora_qwen2_5_7b
+  lora_attn_modules: ['q_proj', 'v_proj', 'output_proj']
+  apply_lora_to_mlp: True
+  apply_lora_to_output: False
+  lora_rank: 64  # higher increases accuracy and memory
+  lora_alpha: 128  # usually alpha=2*rank
+  lora_dropout: 0.0
+
+teacher_model:
+  _component_: torchtune.models.qwen2_5.lora_qwen2_5_14b_instruct
+  lora_attn_modules: ['q_proj', 'v_proj', 'output_proj']
+
+# Tokenizer
+tokenizer:
+  _component_: torchtune.models.qwen2_5.qwen2_5_tokenizer
+  path: /Qwen2.5-7B-Instruct/vocab.json
+  merges_file: /Qwen2.5-7B-Instruct/merges.txt
+  max_seq_len: null
+
+checkpointer:
+  _component_: torchtune.training.FullModelHFCheckpointer
+  checkpoint_dir: ./Qwen2.5-7B-Instruct
+  checkpoint_files: [
+    model-00001-of-00002.safetensors,
+    model-00002-of-00002.safetensors,
+    model-00003-of-00002.safetensors,
+    model-00004-of-00002.safetensors,
+
+  ]
+
+  recipe_checkpoint: null
+  output_dir: ${output_dir}
+  model_type: QWEN2
+resume_from_checkpoint: False
+save_adapter_weights_only: False
+
+# Teacher checkpoint
+teacher_checkpointer:
+  _component_: torchtune.training.FullModelHFCheckpointer
+  checkpoint_dir: ./Qwen2.5-14B-Instruct
+  checkpoint_files: [

```

Comment:
Same comment as above - please use the format_filename param. https://github.com/pytorch/torchtune/blob/e2fd3425e3b86f021c05daadeec8b0421e0e7af0/recipes/configs/llama4/scout_17B_16E_full.yaml#L37

Reply:
done

Reply:
@joecummings I have another new issue, should I open an issue or PR directly?